modules/admin_manual/pages/installation/manual_installation/upgrade_install_phpmyadmin.adoc

```diff
@@ -27,77 +27,159 @@ sudo apt-cache policy phpmyadmin
 
 When using `phpmyadmin`, post login, look at the bottom right side to see the actual version running.
 
-image:installation/manual_installation/phpmyadmin_v4.9.5deb2.png[]
+image::installation/manual_installation/phpmyadmin_v4.9.5deb2.png[width=300]
+
+You will find the binaries of your installation in the directory: `/usr/share/phpmyadmin`
+
+== Upgrade Methods
+
+=== PPA Upgrade
+
+// see: https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu
+
+You can upgrade `phpmyadmin` most easiest to a later, but also most likely not to the latest version when adding a PPA provided by the phpmyadmin team. To do so, type the following commands:
+
+[source,bash]
+----
+sudo add-apt-repository ppa:phpmyadmin/ppa
+----
+ 
+[source,bash]
+----
+sudo apt update
+----
+
+[source,bash]
+----
+sudo apt install phpmyadmin
+----
+
+Note, if you do not want to continue with the PPA solution, just remove it by invoking following command. This will keep your existing phpmyadmin installation, but will not update it anymore:
```

Comment:
```suggestion
Note: if you do not want to continue with the PPA solution, just remove it by invoking the following command. This will keep your existing phpmyadmin installation, but will not update it anymore:
```